.circleci/config.yml

```
@@ -15,7 +15,7 @@ references:
     run: {name: 'Disable SSH timeout', command: 'echo "ServerAliveInterval 30" >> ~/.ssh/config'}
 
   vendor_bin_add_path: &vendor_bin_add_path
-    run: echo 'export PATH=vendor/bin:node_modules/.bin:$PATH' >> $BASH_ENV
+    run: echo 'export PATH=vendor/bin:node_modules/.bin:$PATH' >> $BASH_ENV && source $BASH_ENV
```

This change should be reverted and the deployment rerun. This was added as part of debugging deployment issues. CircleCI already sources $BASH_ENV before every command: https://support.circleci.com/hc/en-us/articles/360011501714-Source-a-bash-file-before-every-command-

```suggestion
    run: echo 'export PATH=vendor/bin:node_modules/.bin:$PATH' >> $BASH_ENV
```